src/cmake-tools.ts

```diff
@@ -343,32 +377,41 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    * Implementation of `cmake.configure`
    */
   configure(extra_args: string[] = []) {
-    return this._doConfigure(async (consumer) => {
-      const drv = await this._cmakeDriver;
-      return drv.configure(extra_args, consumer);
+    return this._doConfigure(async(consumer) => {
+      const drv = await this.getCMakeDriverInstance();
+      if (drv) {
+        return drv.configure(extra_args, consumer);
+      } else {
+        return -1;
+      }
     });
   }
 
   /**
    * Implementation of `cmake.cleanConfigure()
    */
   cleanConfigure() {
-    return this._doConfigure(async (consumer) => {
-      const drv = await this._cmakeDriver;
-      return drv.cleanConfigure(consumer);
+    return this._doConfigure(async(consumer) => {
+      const drv = await this.getCMakeDriverInstance();
+      if (drv) {
+        return drv.cleanConfigure(consumer);
+      } else {
+        return -1;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve(-1);
```

Reply:
Same as above.